x/group/spec/group.md

```diff
@@ -131,9 +67,8 @@ to authorize messages send back to the router.
 ```go
 type GroupKeeper interface {
   IterateGroupsByMember(member sdk.Address, fn func (group sdk.AccAddress) (stop bool))
-  IterateGroupsByOwner(member sdk.Address, fn func (group sdk.AccAddress) (stop bool))
+  IterateGroupsByAdmin(member sdk.Address, fn func (group sdk.AccAddress) (stop bool))
```

Comment:
This part was intentional. I didn't define a group struct because I don't want to return all members at once. See how it is in the changed spec and we can discuss if maybe there should be a `GroupMetadata` struct like this:

```go
type GroupMetadata struct {
  Group GroupID
  TotalPower sdk.Int
  Description string
  Admin sdk.AccAddress
}
```

Reply:
Ahh... Okay, then can you mark this in the spec as "not yet defined", but make sure some comment is there, not just an oversight.